memory/memkind_kmem_allocator.cc

```diff
@@ -0,0 +1,33 @@
+/*
+* Copyright (c) 2019 Intel Corporation
+*
+* SPDX-License-Identifier: Apache-2.0
+*
+*/
```

Comment:
RocksDB supports GPLv2 + Apache v2 dual license. Having some source code with only one of them confuses people and will make adaption more complicated for potential users. I would be reluctant to accept source file like this. Ideally, you give the copyright to RocksDB too. I don't think these source files contain any IP worth protecting so hopefully it's easy for you to do it.

Reply:
Headers are updated to include dual license and Facebook copyright.